src/_includes/header-docs.html

```diff
@@ -2,18 +2,9 @@
 
 <header class="siteHeader2 siteHeader--docs">
   <div class="siteHeader-logoAndMenuButton">
-    {% if page.layout == 'landing' %}
```

Comment:
@lianmakesthings's [comment](https://github.com/tilt-dev/tilt.build/pull/1040#discussion_r789619500) about these silly obsolete lines inspired this grab-bag PR!